docs/products/kafka/concepts/auth-types.rst

```diff
@@ -0,0 +1,73 @@
+Authentication types
+======================
+
+It is strongly recommended to use modern encryption protocols to protect data in transit sent to and from Apache Kafka.
+You can use one of the available options in **Aiven for Apache Kafka** services:
+
+Transport Layer Security
+------------------------
+
+**Transport Layer Security (TLS)**, also known as Secure Sockets
+Layer (SSL), is an established standard for securing internet traffic. This method
+relies on a certificate that is provided by a Certificate
+Authority (for example, `letsencrypt.org <http://letsencrypt.org>`_ ) for your domain.
+With this certificate and the right technical setup, you can use
+your domain to encrypt the traffic to your service.
+
+By default Aiven enables TLS encryption for all  **Aiven for Apache Kafka** services and helps with the application, renewal, and configuration of certificates.
+
+There are two ways you can use TLS:
+
+#. **TLS encryption** : your Apache Kafka client validates the certificate for
+   your Apache Kafka broker.
+
+#. **TLS authentication** : your Apache Kafka client validates the certificate
+   for your Apache Kafka broker and your broker validates the certificate
+   for your client.
+
+
+Simple Authentication and Security Layer
+----------------------------------------
+
+**Simple Authentication and Security Layer (SASL)** acts as a layer that
```

Comment:
I don't think we should mention Kerberos here - it could be misleading since we do not support that as an authentication type (SASL/GSSAPI).

Reply:
Removed